apis/eventstream/index.yaml

```diff
@@ -0,0 +1,73 @@
+              value: |
+                event: head\n
+                data: "{\"slot\": \"10\", \"block\": \"0x9a2fefd2fdb57f74993c7780ea5b9030d2897b615b89f808011ca5aebed54eaf\", \"state\":\"0x600e852a08c1200654ddf11025f1ceacb3c2e74bdd5c630cde0838b2591b69f9\", \"epoch_transition\": false}"\n
+                \n
+            block:
+              description: The node has received a block (from P2P or API)
+              value: |
+                event: block\n
+                data: "{\"slot\": \"10\", \"block\": \"0x9a2fefd2fdb57f74993c7780ea5b9030d2897b615b89f808011ca5aebed54eaf\"}"\n
+                \n
+            attestation:
+              description: The node has received an attestation (from P2P or API)
+              value: |
+                event: attestation\n
+                data: "{\n      \"aggregation_bits\": \"0x01\",\n      \"signature\": \"0x1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505cc411d61252fb6cb3fa0017b679f8bb2305b26a285fa2737f175668d0dff91cc1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505\",\n      \"data\": {\n        \"slot\": \"1\",\n        \"index\": \"1\",\n        \"beacon_block_root\": \"0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2\",\n        \"source\": {\n          \"epoch\": \"1\",\n          \"root\": \"0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2\"\n        },\n        \"target\": {\n          \"epoch\": \"1\",\n          \"root\": \"0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2\"\n        }\n      }\n    }"\n
+                \n
+            voluntary_exit:
+              description: The node has received a voluntary exit (from P2P or API)
+              value: |
+                event: voluntary_exit\n
+                data: "{\n      \"message\": {\n        \"epoch\": \"1\",\n        \"validator_index\": \"1\"\n      },\n      \"signature\": \"0x1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505cc411d61252fb6cb3fa0017b679f8bb2305b26a285fa2737f175668d0dff91cc1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505\"\n    }"\n
+                \n
+            finalized_checkpoint:
+              description: Finalized checkpoint has been updated
+              value: |
+                event: finalized_checkpoint\n
+                data: "{\"block\":\"0x9a2fefd2fdb57f74993c7780ea5b9030d2897b615b89f808011ca5aebed54eaf\", \"state\": \"0x600e852a08c1200654ddf11025f1ceacb3c2e74bdd5c630cde0838b2591b69f9\", \"epoch\": \"2\" }"\n
+                \n
+            chain_reorg:
+              description: The node has reorganized its chain
+              value: |
+                event: chain_reorg\n
+                data: "{\"slot\":\"200\", \"depth\": \"50\", \"old_head_state\":\"0x9a2fefd2fdb57f74993c7780ea5b9030d2897b615b89f808011ca5aebed54eaf\", \"state\": \"0x600e852a08c1200654ddf11025f1ceacb3c2e74bdd5c630cde0838b2591b69f9\", \"epoch\": \"2\" }"\n
```

Comment:
You have `old_head_state` and `state` here, which seems a bit weird.  Would be better for them to either be `old_state` and `state`, or `old_head_state` and `head_state`.

Reply:
Didn't you define those fields in WG Proposal sheet? :smile: 
Maybe old_head_state, new_head_state?

Curious as to why state roots and not block roots?

Reply:
@mcdee ^^

Reply:
Sorry was going to look back to where I picked state root rather than block root, but I can't find it so either my detective skills are very poor or I made it up; either way I'll take the blame :laughing: 

State roots allow us to fetch items from /beacon/states directly, which seems to be a logical next step when the chain has reorganized.  But if you think that block roots are a good idea in addition we could add them in.

And yes, `old_head_state` and `new_head state` are nicely explicit so let's go with them.

Reply:
I'm gonna add block roots since full beacon states are debug endpoint and I don't see I way to determine block heads from current state endpoints?